config/authorization/config.ex

```diff
@@ -436,6 +443,18 @@ defmodule Acl.UserGroups.Config do
         ]
       },
 
+      # Read-write access to public graph for Themis export types
+      %GroupSpec{
+        name: "themis-export",
+        useage: [:read, :write, :read_for_write],
```

Comment:
Write/read_for_write access is not required afaik since the export service is a background service using sudo queries to insert data. I think the `themis_export_types()` can just be added to the existing public read configuration.